modules/oneVideoBidAdapter.md

```diff
@@ -9,44 +9,76 @@
 Connects to One Video demand source to fetch bids.
 
 
-# Test Parameters
+# Test Parameters for Video
 ```
     var adUnits = [
-	{
-      code: 'video1',
-        sizes: [640,480],
-        mediaTypes: {
-          video: {
-            context: "instream"
-          }
-        },
-        bids: [
-          {
-            bidder: 'oneVideo',
-            params: {
-              video: {
-                playerWidth: 480,
-                playerHeight: 640,
-                mimes: ['video/mp4', 'application/javascript'],
-                protocols: [2,5],
-                api: [2],
-                position: 1,
-                delivery: [2],
-                playbackmethod: [1,5],
-                placement: 123,
-                sid: <scpid>,
-                rewarded: 1
+        {
+            code: 'video1',
+              sizes: [640,480],
+              mediaTypes: {
+                video: {
+                  context: "instream"
+                }
               },
-             },
-              site: {
-                id: 1,
-                page: 'http://abhi12345.com',
-                referrer: 'http://abhi12345.com'
+              bids: [
+                {
+                  bidder: 'oneVideo',
+                  params: {
+                    video: {
+                      playerWidth: 480,
+                      playerHeight: 640,
+                      mimes: ['video/mp4', 'application/javascript'],
+                      protocols: [2,5],
+                      api: [1],
+                      position: 1,
+                      delivery: [2],
+                      playbackmethod: [1,5],
+                      sid: <scpid>,
+                      rewarded: 1
+                    },
+                    site: {
+                      id: 1,
+                      page: 'http://abhi12345.com',
+                      referrer: 'http://abhi12345.com'
+                    },
+                    pubId: 'brxd'
+                  }
+                }
+            ]
+       }
+]
+```
+# Test Parameters for banner request
+```
+    var adUnits = [
+        {
+            code: 'video1',
+              sizes: [640,480],
+              mediaTypes: {
+                video: {
+                  context: "instream"
+                }
               },
-              pubId: 'brxd'
-            }
-          }
-        ]
-    }
-	];
+              bids: [
+                {
+                  bidder: 'oneVideo',
+                  params: {
+                    video: {
+                      playerWidth: 480,
+                      playerHeight: 640,
+                      mimes: ['video/mp4', 'application/javascript'],
+                      position: 1,
+                      display: 1
```

Comment:
So for a `banner` ad unit you expect the `video.context` to be `instream`?

Do you expect this "banner" adUnit to render inside of some player?

What exactly is the use case, is this supposed to be `outstream`?

Also, if you are testing with a banner customer, why is "banner" not a `supportedMediaType` on this [line](https://github.com/prebid/Prebid.js/pull/4344/files#diff-31ef8e5337b272996bcfbcdb1a671351R11) 

Looks like your adServer responds with html. How can I get your test creative to render on a test page?

I am getting a lot of errors trying to render it using the standard Prebid Universal Creative.

Reply:
@robertrmartinez yes I expect video.context as instream and for outstream right now we are not supporting. yes it should render in some player which will be sent by adserver.
I am pasting the test page below in which adserver sends the XML in bid.adm param 

`<html>
<div class="centrePlayer" align="center"><script>var o2playerSettings = {aegOverride: {type: "Preroll",vastXml: "<?xml version=\"1.0\" encoding=\"UTF-8\"?><VAST version=\"2.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:noNamespaceSchemaLocation=\"oxml.xsd\"><Ad id=\"a1174066_202836126\"><Wrapper><AdSystem version=\"1.0\">Adap.tv</AdSystem><VASTAdTagURI><![CDATA[https://bid.g.doubleclick.net/dbm/vast?dbm_c=AKAmf-DX-Yo1zAoE8LYhX1eFSj3nr1UaUOWqvka-xpuwZtcl_9GE2qi19IG3APZc1eg7x08wXklKJGkZ9Mi3Wl2VHpsbtaaWVA&dbm_d=AKAmf-BSyUPS4UILQocG1UD3btpiBRp30hlRexnACH0QZFh9mf5HSOZ43ft2eoTdXML49L-qsgQr-c4mCV1ja6KYIroJt_PepPM-XkVmrGii3LwDyhFZ3xB5oAA7rGVaw0YCsCsssIxfPYHYRJS7k4DysgT0fgOkkMfC0_qS_4J3ve-avy209mrlemSlpu6eX-BzLk_jobp7x9UjufWKpgIjo6N_DTm7GNi3GdK3yEUC5pL881REyfr2bgmA3Vye5qCessxiWLg7hmThIf2T4imaH1lR2Wg16U1_wQQq_q3GCouF5LTziMkZfLn-068uRwxDFz9AutOH3vsd9uHt4-9ylYwnmJxr0eudQGGKBD43rufPdYNzUJrsW22PQbZZfJBxrIZOCB_jwzPb5sdtIWRK_Qc-7ofIdJZvHTj_qBF1rW270zXZUvncRt8WsB4ZWTWY7S8AW5m-YafXZDjW9Trz5K2AfT6MpJI22_NejeEeb1Iqaa5KCxNIjaYZoeW2RraEsVgTrJc8PW7TVqO6TohdhP4U57ljOEPI654ui4AwbXMZrILXsyGt9CRIT9K-cbWWK1ZWtpgirsIEmMGP-3JaHdH5IxuK31lobcH79R5LBRh7yPn6zjx4MqC0S6YJGuLmgdE1K-4O5d31j55fUEufndGdP2J1vdwGo24aaCntutZnlEBa8JW_XMay85uIA-9PnnlOsfKrD7J2basrdkj3-w274c7sS5KpIh1kq-r-cqSW65myBtNk2vBJJkwZDD-uzso-07psYE-P2CbErViSH3C4pmZ8F9W7T8WSwO4WxNlvt3r3ya3cIeSPHctVxX0hzhUS4ejXlfMn3varaJxxSTxhwo9c08_vYqNW0PFg5GmnGqve6pg_pcQ8RZzBLW6YWxXGM28bHGRh757o2HeF3zZyGN1tHMyS4lxw7GtMlDuRPg6jhrk2gBhfFubbpZbNYwgKoCwWcrHYDxtSLgL8L2RrgCOmb2mljulAl23VvE-gOyil1q48Gyn7FIqf_K-_u8qbTKDifD_VKpVbXpfGPN4QO0u8mbThAcK_Wp7OOCbJeajgJ_kzb7Ep3GYsmG48PfHqJwm5Sn1ZEnI0OBdFTqwaBlfN7ZMeEl8UBZL0BDSaR-Lpwl2P9uuABnuLxYAlo9Avu_e8lnRSpFqAmZcjfE1RfWQMsWKJlXN4fPWWXlTDeaKK7WuPQhdPxo455sbBQQ8Q-Vgt_uVG45XgSTKyPdmcwyTz_6Vzdwu2YNIPwUD0E-CuCPt-Cn8ocLhFKjyNcMaNYmgHD7kfn8Zjooy3lTJhxD9vDTcuAsqmvZUG5ctoUqGfNswhY3C0rsMW_Ae1OBFwvnAWDSRLtN3p2-qIBluyDcZW7URkJVHdl3rZw-1I4U726KPv_mb8B7YcIIGqadR4Jr6B9O2vH7NCnh9DYteZlG87l4WhwIq46OJmj_lGzkLFTm64Y2YqmH212ovPM6a3DZxNBWkv7Vy8bMvYHp7BPapMkXu0lESZZh1nOJn4gPd5dNtIRWrr-QpitVZJczRnM-o1Ndk5T98uMAOasJmJ8NWH14FPWEaMX5a4wBsasVQJD19bLcyycgCieKLdwHXNImm7-vrMGEvHjjMc6lQJvZCUqFs9vBbQPDLyWkdoJTAFbIx4UUZavw_UFWy_Qvb_Ej5ClBv0eWsZ3VgVKb2YqECdekpaSOGNWqCdDmTg84sa7Yn2KvR4Co2Pfo2-EldzgZCRkmGo0JBWxP2iPSTw_hfIwjRHYcbdqhTu_dS-KwI7OY-QxRy9hwJlhivMkrE-Eb_xEGU011NU12NsrPQ9xqDtgtUK1Szn9EhUqZmo1gK_gmXWmWD0IAipPhjvNr8yb5--6WXg1jvHa9RTmnhBcT8CaUxxqQRQvJ2b3lLaS85W7SAGOleW9e2JUssY5mo7A8LTMIvQ0agxtMZYjTj_6QyFzMA3MjI0UvdM4tjRugK-9P_GJzzFdeJJA7RiPQIn9iCIM5cqbgw92VFO6pSo9bsmbKmxMcnNBkRe3EHrWObD8ttsFVZMp7EB0Ds2V5xy33EPYKyTvIr9tZYnL0HVKj5BJ2fAOWm8992aEqnjZQjW2PjK1GkSGwqOxTxUrOOdD3EvlCPJ3SrDVtqqE_CiG6r9--3OHR3yQlzr1x3tf68p4egc-7HVXdxatfQjPomXv99sTw3KkBYJo4EnIxQOOnM1cbQ-hWdZiaW6Y6tLWRL419U5iPDfmmhQGAV-syYXFWQYbey9-SBWvCKWUQ4INb2ayE9Sps4Oa-rbW6q_ETX7R6TYo81Mrs8jpflr5L0UXepvVRy7DZi5QI2lF7-WLailQU9RtCUFeH0DE7iuoe8GDDjZ5Uyoese9igHdbkiDiCNYycpNkWkoOdLx0FfJ6ejdcEWe0ifUkFizUHB8v58D-WeyQqFRplqlztltycggxo68TVJ8jrxyEGPT7w_zmyJ_APPCuHO_Iyv2uW8ZWOXi6cv0pD-nH_gOkRWowuO6I4PLjJMNI3DS7ffr0ewJ_poR1uLfW3BDy2QMdIDMejZJRRrEaRVcT1zkOir74Og1huVEYIrfAxnFTTsAEgiOsiEeNSiEt6BN5To0N5iEUDh_1FBfuQZkVW58PDkmJcx0645UBWHByXJtaEYX7CtrEf0yR4mKwdqu5sBKdR4oOaM4FgJizXoN-Vj6MIsGOJslnzvuAAkFOAB-WocpapX72gGlDuPDXRT3RRxEezvuBIMz6cJ3wbdsbVxRMlFzaiwpfeCe9vZf5ZTu7R9So2NkZs_WEK1sJ3GMB2JWrpVDueYGcvi3l4QqKzX2TLy5it2CHl7sbxpzx2QVK6om-otaSUOwBLGK74JvDrMCClIKoTzbyTPoFJitrJfqu21_Goa3fy0IGHa-IhRtPSRZl5oKSf80mGptvhcurAG6IdhDPaAJG1-_ybyRYlxJp5LItly585GruhSzdUPBIEWFx-jPZraFWauQZHonr0oXjGyk7fGuqqgBcaXtraXd6NpGfUkACugk&cid=CAASEuRorSnLbF5iBiGzStB3N7AfIA&pr=17:0.0200]]></VASTAdTagURI><Error><![CDATA[http://log.adaptv.advertising.com/log?3a=error&132=3459&4e={lastBid}&34={errNo}&6c=G%2F1cd8HSICTSDB0fmIOGQfNBv%2FOgA%2FqIbkXuVC4-Uq8FRJIwrkDcIVTwGjbIPbFF&59={nF}&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&fa=0&ff=0&b=-2&124=202836126&12a=google.com&14d=28&15c=2545&167=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&137=0&138=0&18d=0&a.cv=1]]></Error><Impression><![CDATA[http://log.adaptv.advertising.com/log?3a=adSuccess&51=AX1rAq5H6R4_&50=ZX4madbHHCc_&mtpbr=DR7cxGttIKg_&d={adSourceGroups}&b=-2&53=ZX4madbHHCc_&2c=yyOZX1NC9jo_&164=AX1rAq5H6R4_&156=ZX4madbHHCc_&157=ZX4madbHHCc_&16b=1174066&16a=AX1rAq5H6R4_&165=AX1rAq5H6R4_&168=AX1rAq5H6R4_&6c=G%2F1cd8HSICTSDB0fmIOGQfNBv%2FOgA%2FqIbkXuVC4-Uq8FRJIwrkDcIVTwGjbIPbFF&28=qUsI3M4M68M_&a8=oOt0lqLFswM_&e5=oOt0lqLFswM_&4b=1174066%2C1166696&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&189=true&e9=auc_pr_b64&e8=MC4wMjA=&97=USD&ec=1&19c=3199c0e9de885a6ca5f0ba65df4c26b87f6e4a3f&f6=iCSArrpvL%2FM_&12e=-1.00&12f=-1.00&130=-1.00&131=-1.00&15b=-1&162=nSD9S7iOK-aaEmbB2QbGtw__&163=5cF-AQrdSok_&fa=0&ff=0&103=qUsI3M4M68M_&102=oOt0lqLFswM_&104=oOt0lqLFswM_&auctionId=5c167f8a-df9c-4d8e-a026-f58d96ffd556&15d=AX1rAq5H6R4_&adSpend=zVOZvN5jJDI_&10f=1571115148&91=ONLINE_VIDEO&121=3&124=202836126&12a=google.com&14d=28&12c=0&12d=11145&132=3459&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&137=0&14e=XkNQzJQe%2F-2uMRdxks2ziQ__&nextSeatbid=0&15c=2545&colo=AS&a.afv=-1&a.dfv=-2&167=1&16d=ab81cb3&16e=gayoncMgHis_&bpTime=0&UIDAdFreqCount=-1&13a=1&139=9a61c675-4802-3219-899e-0c2592284c51&13b=3&18a=2&138=0&18d=0&isVpcInjected=0&bid_floor=1&a.cv=1]]></Impression><Impression><![CDATA[http://b.scorecardresearch.com/b?c1=1&c2=6034979&c3=aolvideo&c4=yahoo.com&c5=090200&c6=1166696]]></Impression><Impression><![CDATA[http://secure-us.imrworldwide.com/cgi-bin/m?ci=us-305284&c6=vc,b01&cg=6186&tl=dav0-%5B0%5D1174066&cc=1&rnd=50872909&c3=st,a]]></Impression><Impression><![CDATA[http://b.scorecardresearch.com/b?c1=1&c2=9545188&c3=&c4=&c5=0100000&rn=123456789012]]></Impression><Impression><![CDATA[http://conversions.adaptv.advertising.com/conversion/wc?adSourceId=1166696&bidId=1174066&afppId=&creativeId=&mediaId=360952&marketplaceId=1&key=aolvideo&a.pvt=0&a.rid=5c167f8a-df9c-4d8e-a026-f58d96ffd556&a.pub_id=&nielsen_devid=&a.platformDevice=ONLINE_VIDEO&eov=50872909]]></Impression><Impression><![CDATA[http://log.adaptv.advertising.com/log?3a=marketTransaction&50=ZX4madbHHCc_&51=AX1rAq5H6R4_&52=AX1rAq5H6R4_&mtpbr=DR7cxGttIKg_&5=1166696&14=1174066&11d=5010921164734968&111=1&6a=-2&6b=-2&99=1&e9=auc_pr_b64&e8=MC4wMjA=&97=USD&f6=iCSArrpvL%2FM_&12e=-1.00&12f=-1.00&130=-1.00&131=-1.00&15b=-1&162=nSD9S7iOK-aaEmbB2QbGtw__&163=5cF-AQrdSok_&138=0&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Impression><Impression><![CDATA[http://adadvisor.net/adscores/g.pixel?sid=9204679687&_p=targusinfo]]></Impression><Impression><![CDATA[http://pixel.quantserve.com/pixel/p-c9d_b-0iR8pjg.gif]]></Impression><Impression><![CDATA[http://pixel.quantserve.com/seg/r;a=p-c9d_b-0iR8pjg;redirect=http://segments.adap.tv/data?p=quantcast-adaptv&type=gif&segment=!qcsegs&add=true]]></Impression><Impression><![CDATA[http://pixel.quantserve.com/seg/r;a=p-c9d_b-0iR8pjg;redirect=http://segments.adap.tv/data?p=quantcast-lal&type=gif&segment=!qcsegs&add=true]]></Impression><Impression><![CDATA[http://adadvisor.net/adscores/g.pixel?sid=9204679687&_p=audpartners]]></Impression><Impression><![CDATA[http://ps.eyeota.net/match?bid=6bib2cv&uid=ASX4137132553786177744]]></Impression><Impression><![CDATA[http://adadvisor.net/adscores/g.pixel?sid=9201748608]]></Impression><Impression><![CDATA[http://adadvisor.net/adscores/g.pixel?sid=9202165158]]></Impression><Impression><![CDATA[http://ib.adnxs.com/getuid?http://seg.a3cloud.net?uid=$UID]]></Impression><Impression><![CDATA[NULL]]></Impression><Impression><![CDATA[https://pr-bh.ybp.yahoo.com/sync/adtech/O262402f3e-2c41-11e8-82fd-0242f150ca96?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://cm.g.doubleclick.net/pixel?google_nid=adaptv_dbm&google_cm&google_sc&_origin=0&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://match.adsrvr.org/track/cmf/generic?ttd_pid=adaptv&ttd_tpi=1&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://sync-tm.everesttech.net/upi/pid/m7y5t93k?gdpr=0&gdpr_consent=&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55986%2Fsync%3Fuid%3D%24%7BUSER_ID%7D%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://image8.pubmatic.com/AdServer/ImgSync?p=156084&gdpr=0&gdpr_consent=&pu=https%3A%2F%2Fimage4.pubmatic.com%2FAdServer%2FSPug%3Fp%3D156084%26gdpr%3DPM_GDPR%26gdpr_consent%3DPM_CONSENT%26pmc%3DPM_PMC%26pr%3Dhttps%253A%252F%252Fpixel.advertising.com%252Fups%252F55983%252Fsync%253F_origin%253D0%2526gdpr%253DPM_GDPR%2526gdpr_consent%253DPM_CONSENT%2526uid%253D%2523PMUID]]></Impression><Impression><![CDATA[https://ad.turn.com/r/cs?pid=18&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://sync.tidaltv.com/adaptv.ashx?gdpr=0&consent=]]></Impression><Impression><![CDATA[https://sync.mathtag.com/sync/img?mt_exid=21&gdpr=0&gdpr_consent=&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55938%2Fsync%3Fuid%3D%5BMM_UUID%5D%26_origin%3D0%26gdpr=0%26gdpr_consent=]]></Impression><Impression><![CDATA[https://secure.adnxs.com/getuid?https%3A%2F%2Fpixel.advertising.com%2Fups%2F55936%2Fsync%3Fuid%3D%24UID%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://x.bidswitch.net/sync?ssp=adaptv&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://track.eyeviewads.com/sync/adaptv?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://sync.1rx.io/usersync2/aol?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://ssum-sec.casalemedia.com/usermatchredir?s=183965&cb=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55940%2Fsync%3Fuid%3D__UID__%26]]></Impression><Impression><![CDATA[https://us-u.openx.net/w/1.0/cm?id=9e0a35ea-c8e3-4b1b-9efa-4af6f54a373e&gdpr=0&gdpr_consent=&r=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55981%2Fsync%3F_origin%3D0%26gdpr%3D0%26gdpr_consent%3D%26uid%3D]]></Impression><Impression><![CDATA[https://dis.criteo.com/dis/usersync.aspx?r=45&p=96&cp=millennial&cu=1&url=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55945%2Fsync%3F_origin%3D0%26uid%3D%40%40CRITEO_USERID%40%40]]></Impression><Impression><![CDATA[https://pixel.quantserve.com/pixel/p-NcBg8UA4xqUFp.gif?idmatch=0&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://um.simpli.fi/match_redirect?gdpr=0&gdpr_consent=&sifi_redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55964%2Fsync%3Fuid%3D%24UID%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://gu.dyntrk.com/adx/adptv/us.php?dynk=176a0l&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://cs.emxdgt.com/um?ssp=aol]]></Impression><Impression><![CDATA[https://p.rfihub.com/cm?in=1&pub=758&gdpr=0&gdpr_consent=&forward=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55856%2Fsync%3Fuid%3D%7Buserid%7D%26_origin%3D0%26gdpr=0%26gdpr_consent=]]></Impression><Impression><![CDATA[https://atp.mxptint.net/sn.ashx?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://pm.w55c.net/ping_match.gif?gdpr=0&gdpr_consent=&st=ONEMOBILE&rurl=https%3A%2F%2Fpixel.advertising.com%2Fups%2F56554%2Fsync%3Fuid%3D_wfivefivec_%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://match.prod.bidr.io/cookie-sync/at?]]></Impression><Impression><![CDATA[https://pixel-sync.sitescout.com/dmp/pixelSync?nid=88&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://rtb.adentifi.com/CookieSyncAdapTV?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://aol-match.dotomi.com/match/bounce/current?networkId=60&version=1&nuid=O262402f3e-2c41-11e8-82fd-0242f150ca96&gdpr=0&gdpr_consent=&rurl=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55853%2Fsync%3Fuid%3D%24UID%26_origin%3D0]]></Impression><Impression><![CDATA[https://a.tribalfusion.com/i.match?p=b17&u={{APID}}&redirect=https%3A%2F%2Fpixel.advertising.com%2Fups%2F57628%2Fsync%3F_origin%3D0%26redir%3Dtrue%26uid%3D%24TF_USER_ID_ENC%24]]></Impression><Impression><![CDATA[https://dt-secure.videohub.tv/v1/usync/adaptv?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://u.acuityplatform.com/us?tpid=121&tpuid=O262402f3e-2c41-11e8-82fd-0242f150ca96&gdpr=0&gdprc=&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55950%2Fsync%3Fuid%3D%24UID%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://sync.bfmio.com/syncb?pid=127]]></Impression><Impression><![CDATA[https://d5p.de17a.com/getuid/adaptv?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://d5p.de17a.com/getuid/aol?gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://idpix.media6degrees.com/orbserv/hbpix?pixId=4098&pcv=42]]></Impression><Impression><![CDATA[https://tags.bluekai.com/site/14143?id=ASX4137132553786177744]]></Impression><Impression><![CDATA[https://loadm.exelator.com/load/?p=204&g=370&buid=ASX4137132553786177744&j=0]]></Impression><Impression><![CDATA[https://dps.bing.com/AI/Api/v1/UserRest.svc/Provider/178B9C68-45E7-4F9B-B33D-240F152A03C1/User/ASX4137132553786177744/gif?meta=Adap]]></Impression><Impression><![CDATA[https://sync.search.spotxchange.com/partner?adv_id=6920&uid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://dpm.demdex.net/ibs:dpid=832&dpuuid=ASX4137132553786177744&redir=]]></Impression><Impression><![CDATA[https://ads.stickyadstv.com/user-registering?dataProviderId=145&userId=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://rp.gwallet.com/r1/cm/p52r]]></Impression><Impression><![CDATA[https://d.turn.com/r/du/id/L2NzaWQvMS9tcGlkLzMzOTA0ODEz/mpuid/ASX4137132553786177744]]></Impression><Impression><![CDATA[https://ps.eyeota.net/match?bid=6bib2cv&uid=ASX4137132553786177744]]></Impression><Impression><![CDATA[https://sy.eu.angsrvr.com/sync?type=host&dsp=12&dspuuid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://ad.sxp.smartclip.net/sync?type=red&dsp=29]]></Impression><Impression><![CDATA[https://geo-um.btrll.com/v1/map/8ce6ac1ff5f5cd2f/4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96/]]></Impression><Impression><![CDATA[https://usersync.videoamp.com/usersync?gdpr=0&gdpr_consent=&partner_id=5797361&partner_user_id=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&redirect=https%3A%2F%2Fpixel.advertising.com%2Fups%2F56549%2Fsync%3Fuid%3D%24UID%26_origin%3D0%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[https://cks.yumenetworks.com/cookiesync/pixel/3/ymvw/50872909.gif?puuid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://pixel.rubiconproject.com/exchange/sync.php?p=oath&gdpr=0&gdpr_consent=]]></Impression><Impression><![CDATA[https://bh.contextweb.com/bh/rtset?pid=561707&ev=1&gdpr=0&gdpr_consent=&rurl=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55972%2Fsync%3Fuid%3D%25%25VGUID%25%25%26_origin%3D0]]></Impression><Impression><![CDATA[https://aa.agkn.com/adscores/g.pixel?sid=9212285868&aid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://adserver.video/sync/03635d2e5423be5c297a9b6f812b727e/?action=in&uid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.teads.tv/um?eid=18&uid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://px.adcloudiq.com/ap?AUID=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://ads.scorecardresearch.com/p?c1=9&c2=1000009&c3=2&cs_xi=O262402f3e-2c41-11e8-82fd-0242f150ca96&rn=87599618]]></Impression><Impression><![CDATA[https://ads.stickyadstv.com/user-registering?dataProviderId=449&userId=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://targeting.unrulymedia.com/usermatch/aolone/4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://onevideosync.uplynk.com/usync?key=onevideo&comboId=O262402f3e-2c41-11e8-82fd-0242f150ca96&gdpr=0&euconsent=&redirect=true]]></Impression><Impression><![CDATA[https://dsp.adfarm1.adition.com/cookie/?ssp=24&gdpr=0&gdpr_consent=&]]></Impression><Impression><![CDATA[https://pixel.advertising.com/ups/58156/sync?_origin=1&uid=[DSP_USER_ID]&gdpr=[GDPR_JURISDICTION]&gdpr_consent=[consent_str_based_on_iab_specification]]]></Impression><Impression><![CDATA[https://trc.taboola.com/sg/oathvideo-network/1/rtb-h?taboola_hm=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.springserve.com/usersync?aid=759&uuid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://pixel.advertising.com/ups/58163/sync?gdpr=&gdpr_consent=&_origin=0&redir=true]]></Impression><Impression><![CDATA[https://live.sekindo.com/live/liveCS.php?source=external&pixel=&advId=25462&advUuid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://ssp.playbuzz.com/sync?tp_id=8&tp_uid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.technoratimedia.com/services?srv=cs&pid=52&uid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.adtelligent.com/csync?t=a&ep=284006&extuid={{APID}}]]></Impression><Impression><![CDATA[https://dmp.truoptik.com/b597f5a183506ddb/sync.gif?dm=117.104.190.44]]></Impression><Impression><![CDATA[https://tracker.konduit.me/sync?vendorName=oath&cid=5978c36a9140d272a115bdb6&vendorUserIdKey=APID&vendorUserIdValue=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://cs.rtb-seller.com/cs?aid=11554&id=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.aniview.com/cookiesyncendpoint?biddername=25&key=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.spotim.market/csync?t=a&ep=290146&extuid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://openrtb.cootlogix.com/api/cookie?partnerId=ob%20-%20adaptv%20A&userId=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://onetag-sys.com/sync/i,39/O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://cm.g.doubleclick.net/pixel?google_nid=oath_eb&google_hm=TzI2MjQwMmYzZS0yYzQxLTExZTgtODJmZC0wMjQyZjE1MGNhOTY%3D]]></Impression><Impression><![CDATA[https://cookiesync.slyngshot.io/sync/dsp?uid=O262402f3e-2c41-11e8-82fd-0242f150ca96&partner=135&zone=[zoneid]]]></Impression><Impression><![CDATA[https://verizon.adhaven.com/bid-engine/cs/f68e46bc869e960e/v1?rd=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55960%2Fsync%3Fuid%3D%24UID%26_origin%3D0]]></Impression><Impression><![CDATA[https://sync.aniview.com/cookiesyncendpoint?biddername=25&key=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://public-prod-dspcookiematching.dmxleo.com/dspreply?dspId=42&dspUserId=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://bidder.9mediaonline.com/pix/4099/dcm?dusr=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.aniview.com/cookiesyncendpoint?biddername=25&key=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.sync.viewdeos.com/csync?t=a&ep=302179&extuid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[https://sync.adotmob.com/cookie/nexage?gdpr=0&gdpr_consent=&r=https%3A%2F%2Fpixel.advertising.com%2Fups%2F57773%2Fsync%3F_origin%3D0%26uid%3D%24{amob_user_id}%26gdpr%3D0%26gdpr_consent%3D]]></Impression><Impression><![CDATA[http://a01.korrelate.net/a/e/d2us.ads?r=123456789&et=us&ago=127&ao=534&c_uid[aol]=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Impression><![CDATA[http://ads.adap.tv/copy_cookies?]]></Impression><Impression><![CDATA[http://ups.analytics.yahoo.com/ups/58217/occ?apid=O262402f3e-2c41-11e8-82fd-0242f150ca96]]></Impression><Creatives><Creative id=\"6186-202836126\" adId=\"1174066\"><Linear><TrackingEvents><Tracking event=\"start\"><![CDATA[http://log.adaptv.advertising.com/log?3a=progressDisplay0&132=3459&12a=google.com&14d=28&15c=2545&124=202836126&12d=11145&167=1&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)&a.cv=1]]></Tracking><Tracking event=\"start\"><![CDATA[http://b.scorecardresearch.com/p?c1=2&c2=6000006&ns_ap_sv=2.1511.10&ns_type=hidden&ns_st_it=a&ns_st_sv=4.0.0&ns_st_ad=1&ns_st_sq=1&ns_st_id=3578558589746455922&ns_st_ec=1&ns_st_cn=1&ns_st_ev=play&ns_st_ct=va&ns_st_cl=0&ns_st_pt=0&c3=*null&c4=*null&c6=*null&ns_ts=*null]]></Tracking><Tracking event=\"expand\"><![CDATA[http://log.adaptv.advertising.com/log?3a=expandedChange&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Tracking><Tracking event=\"firstQuartile\"><![CDATA[http://log.adaptv.advertising.com/log?3a=progressDisplay25&132=3459&12a=google.com&14d=28&15c=2545&124=202836126&167=1&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&18a=2&12d=11145&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)&a.cv=1]]></Tracking><Tracking event=\"midpoint\"><![CDATA[http://log.adaptv.advertising.com/log?3a=progressDisplay50&132=3459&12a=google.com&14d=28&15c=2545&124=202836126&167=1&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&18a=2&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)&a.cv=1]]></Tracking><Tracking event=\"thirdQuartile\"><![CDATA[http://log.adaptv.advertising.com/log?3a=progressDisplay75&132=3459&12a=google.com&14d=28&15c=2545&124=202836126&167=1&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&18a=2&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)&a.cv=1]]></Tracking><Tracking event=\"complete\"><![CDATA[http://log.adaptv.advertising.com/log?3a=progressDisplay100&132=3459&12a=google.com&14d=28&15c=2545&124=202836126&167=1&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&91=ONLINE_VIDEO&f6=iCSArrpvL%2FM_&12e=-1.00&12f=-1.00&130=-1.00&131=-1.00&15b=-1&162=nSD9S7iOK-aaEmbB2QbGtw__&163=5cF-AQrdSok_&18a=2&138=0&18d=0&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)&a.cv=1]]></Tracking><Tracking event=\"complete\"><![CDATA[http://b.scorecardresearch.com/p?c1=2&c2=6000006&ns_ap_sv=2.1511.10&ns_type=hidden&ns_st_it=a&ns_st_sv=4.0.0&ns_st_ad=1&ns_st_sq=1&ns_st_id=3578558589746455922&ns_st_ec=2&ns_st_cn=1&ns_st_ev=end&ns_st_ct=va&ns_st_cl=0&ns_st_pt=0&c3=*null&c4=*null&c6=*null&ns_ts=*null]]></Tracking><Tracking event=\"acceptInvitation\"><![CDATA[http://log.adaptv.advertising.com/log?3a=acceptInvitation&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Tracking><Tracking event=\"pause\"><![CDATA[http://log.adaptv.advertising.com/log?3a=paused&12a=google.com&14d=28&124=202836126&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Tracking><Tracking event=\"resume\"><![CDATA[http://log.adaptv.advertising.com/log?3a=playing&12a=google.com&14d=28&124=202836126&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Tracking></TrackingEvents><VideoClicks><ClickTracking><![CDATA[http://log.adaptv.advertising.com/log?3a=click&d3={a.cPos}&6c=G%2F1cd8HSICTSDB0fmIOGQfNBv%2FOgA%2FqIbkXuVC4-Uq8FRJIwrkDcIVTwGjbIPbFF&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&auctionId=5c167f8a-df9c-4d8e-a026-f58d96ffd556&91=ONLINE_VIDEO&12a=google.com&14d=28&15c=2545&124=202836126&132=3459&167=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&138=0&18d=0&f6=iCSArrpvL%2FM_&12e=-1.00&12f=-1.00&130=-1.00&131=-1.00&15b=-1&162=nSD9S7iOK-aaEmbB2QbGtw__&163=5cF-AQrdSok_&a.cv=1]]></ClickTracking><ClickTracking><![CDATA[http://conversions.adaptv.advertising.com/conversion/wc?adSourceId=1166696&bidId=1174066&afppId=&creativeId=&mediaId=360952&marketplaceId=1&key=aolvideo&a.pvt=0&a.rid=5c167f8a-df9c-4d8e-a026-f58d96ffd556&a.pub_id=&nielsen_devid=&a.platformDevice=ONLINE_VIDEO&eov=50872909&a.click=true]]></ClickTracking></VideoClicks></Linear></Creative></Creatives><Extensions><Extension type=\"OneSource creative\"><CreativeId><![CDATA[4294967295]]></CreativeId></Extension><Extension type=\"ad_chromes\"><AdChromes><AdChrome type=\"MoatChrome\"><![CDATA[classpath://tv.adap.adchrome.MoatChrome]]><orgId><![CDATA[]]></orgId><buyerId><![CDATA[6186]]></buyerId><campaignId><![CDATA[0]]></campaignId><adId><![CDATA[1174066]]></adId><marketplaceId><![CDATA[1]]></marketplaceId><pageUrl><![CDATA[yahoo.com]]></pageUrl><duration><![CDATA[00:00:15.000]]></duration><params><![CDATA[&creativeId=&sellerDealId=&doubleAuction=&adSourceId=1166696&bidId=1174066&afppId=&adSourceMediaId=5010921164734968&adSpotId=&pet=&pod=-2&position=-2&marketplaceId=1&optout=0&adPlanId=-2&adaptag=&key=aolvideo&buyerId=6186&campaignId=0&pageUrl=www.yahoo.com%2F&adapDetD=[URL]&sellRepD=yahoo.com&urlDetMeth=&targDSellRep=1&mediaId=360952&zid=&url=&id=&duration=&a.geostrings=&uid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&uid_type=uuid&apid=O262402f3e-2c41-11e8-82fd-0242f150ca96&pid=&is_new_user=0&is_proxy_id=0&htmlEnabled=false&width=&height=&pub.uid=&context=&categories=&sessionId=&serverRev=910140439&playerRev={playerRev}&a.rid=5c167f8a-df9c-4d8e-a026-f58d96ffd556&a.rtime=1571115148466&a.beid=&a.cluster=0&rtype=openrtb&ext_id=OneMDisplay&a.ssc=1&a.asn=ip-10-49-130-196&a.profile_id=1&p.vw.viewable=-1&p.vw.viewableOpportunity=-1&p.vw.viewableHistorical=84.58&p.vw.psize=3&a.sdk=&a.sdkType=&pi.sdk=&pi.sdkType=&a.appReq=0&a.platformDevice=ONLINE_VIDEO&ipAddressOverride=117.104.190.44&a.platformOs=Mac+OS+X&p.vw.active=-1&a.rtbexch=23&pi.sideview=&pi.flashonpage=&pi.mvoa=&pi.avoa=&pi.sound=&pi.autoInitiation=&esVr=0&esMvmr=85&sadVr=0&sadMvmr=83&p.vw.geometric=&p.vw.framerate=&gdpr=0&euconsent=&fnms=&seqAuc=&cdr=&user_agent_source=3&pubContentTopics=&ipSrc=5&a.pub_id=&device_id_status=&consent_given=1&a.ts=0&a.adSeq={adSeq}&isHttps=0&pubSettingId=&pi.psize=[P_VW_SIZE]&advertiser_id=&pageDomain=yahoo.com&bidType=2&eov=50872909]]></params></AdChrome><AdChrome><![CDATA[classpath://tv.adap.adchrome.TelemetryChrome]]><bidId>1174066</bidId><adSourceMediaId>5010921164734968</adSourceMediaId><marketplaceId>1</marketplaceId><pageUrl>www.yahoo.com%2F</pageUrl><rid>5c167f8a-df9c-4d8e-a026-f58d96ffd556</rid></AdChrome></AdChromes></Extension><Extension type=\"adaptv_iab_viewable_beacons\"><Beacon type=\"iab_viewable\"><![CDATA[http://log.adaptv.advertising.com/log?3a=iabViewable&f6=iCSArrpvL%2FM_&12e=-1.00&12f=-1.00&130=-1.00&131=-1.00&15b=-1&162=nSD9S7iOK-aaEmbB2QbGtw__&163=5cF-AQrdSok_&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Beacon><Beacon type=\"iab_detection_started\"><![CDATA[http://log.adaptv.advertising.com/log?3a=iabDetectionStarted&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Beacon><Beacon type=\"iab_detection_failed\"><![CDATA[http://log.adaptv.advertising.com/log?3a=iabDetectionFailed&5=1166696&14=1174066&11d=5010921164734968&6a=-2&6b=-2&4f=1&optout=0&3=-2&5c=aolvideo&5b=6186&18=0&2e=www.yahoo.com%2F&2f=[URL]&30=yahoo.com&32=1&fd=360952&80=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&118=uuid&f8=O262402f3e-2c41-11e8-82fd-0242f150ca96&171=0&190=0&42=false&77=910140439&67={playerRev}&d6=5c167f8a-df9c-4d8e-a026-f58d96ffd556&19d=1571115148466&bf=0&74=openrtb&ed=OneMDisplay&d5=1&d8=ip-10-49-130-196&ae=1&8e=-1&f0=-1&161=84.58&68=3&c4=0&91=ONLINE_VIDEO&45=117.104.190.44&ee=Mac+OS+X&b5=-1&101=23&14b=0&14c=85&152=0&153=83&180=0&197=3&19e=5&195=1&120=0&100={adSeq}&112=0&13d=[P_VW_SIZE]&64=yahoo.com&18a=2&33=50872909&a.cv=1]]></Beacon></Extension><Extension type=\"ViewabilityRules\"><viewabilityDefVideo3P><rule t=\"1\" d=\"2\" c=\"1\" p=\"50\" a=\"0\"></rule><rule t=\"2\" d=\"15\" c=\"0\" p=\"100\" a=\"0\"></rule></viewabilityDefVideo3P></Extension><Extension type=\"Viewability3P\"><Beacon type=\"VIDEO_VIEW_3P\"><![CDATA[http://log.adaptv.advertising.com/log?event=VIDEO_VIEW_3P&creativeId=&sellerDealId=&aud_demo_in_target=iCSArrpvL%2FM_&convr=-1.00&ctr=-1.00&cr=-1.00&vr=-1.00&modelViewRates=&viewModelId=-1&mtbc=nSD9S7iOK-aaEmbB2QbGtw__&mtbt=5cF-AQrdSok_&adSourceId=1166696&bidId=1174066&afppId=&adSourceMediaId=5010921164734968&adSpotId=&pet=&pod=-2&position=-2&marketplaceId=1&optout=0&adPlanId=-2&adaptag=&key=aolvideo&buyerId=6186&campaignId=0&pageUrl=www.yahoo.com%2F&adapDetD=[URL]&sellRepD=yahoo.com&urlDetMeth=&targDSellRep=1&mediaId=360952&zid=&url=&id=&duration=&a.geostrings=&uid=4137132553786177744-O262402f3e-2c41-11e8-82fd-0242f150ca96&uid_type=uuid&apid=O262402f3e-2c41-11e8-82fd-0242f150ca96&pid=&is_new_user=0&is_proxy_id=0&htmlEnabled=false&width=&height=&pub.uid=&context=&categories=&sessionId=&serverRev=910140439&playerRev={playerRev}&a.rid=5c167f8a-df9c-4d8e-a026-f58d96ffd556&a.rtime=1571115148466&a.beid=&a.cluster=0&rtype=openrtb&ext_id=OneMDisplay&a.ssc=1&a.asn=ip-10-49-130-196&a.profile_id=1&p.vw.viewable=-1&p.vw.viewableOpportunity=-1&p.vw.viewableHistorical=84.58&p.vw.psize=3&a.sdk=&a.sdkType=&pi.sdk=&pi.sdkType=&a.appReq=0&a.platformDevice=ONLINE_VIDEO&ipAddressOverride=117.104.190.44&a.platformOs=Mac+OS+X&p.vw.active=-1&a.rtbexch=23&pi.sideview=&pi.flashonpage=&pi.mvoa=&pi.avoa=&pi.sound=&pi.autoInitiation=&esVr=0&esMvmr=85&sadVr=0&sadMvmr=83&p.vw.geometric=&p.vw.framerate=&gdpr=0&euconsent=&fnms=&seqAuc=&cdr=&user_agent_source=3&pubContentTopics=&ipSrc=5&a.pub_id=&device_id_status=&consent_given=1&a.ts=0&a.adSeq={adSeq}&isHttps=0&pubSettingId=&pi.psize=[P_VW_SIZE]&advertiser_id=&pageDomain=yahoo.com&bidType=2&eov=50872909&doubleAuction=&a.abMeth=11145&vt=$(V_VIEW_TYPE)&vln=$(V_VIDEO_LEN)&vsa=$(V_SKIP_AVAIL)&va=$(V_AUTOPLAYED)&vph=$(V_PLAYER_HEIGHT)&vpw=$(V_PLAYER_WIDTH)&ve=$(V_EXPANDED)&vpi=$(V_VIEW_INFO)&api=$(V_AUD_INFO)&atv=$(V_AUD_TIME_INVIEW_100)&ivt=$(V_MOAT_IVT)&vag=$(V_TIME_INVIEW_50)&via=$(V_TIME_INVIEW_50_MAX_CONTINUOUS)&vhj=$(V_IS_INVIEW_101_HALFTIME)]]></Beacon></Extension><Extension type=\"ad_duration\"><![CDATA[00:00:15.000]]></Extension><Extension type=\"one_video_adomain\"><![CDATA[google.com]]></Extension></Extensions></Wrapper></Ad></VAST>"
},playerSizeOverride: {playerWidth: 300,playerHeight: 250}};</script><div style="width:300px;height:250px;background-color:#f5f5f5;"><div id="5bb26ab96614d13012e1fbc8" class="vdb_player vdb_5bb26ab96614d13012e1fbc85bb269bcc1a5ae2480e64380"><script type="text/javascript" src="https://delivery.vidible.tv/jsonp/bid=5d433a90da3c3300018eb6aa/pid=5bb26ab96614d13012e1fbc8/5bb269bcc1a5ae2480e64380.js" ></script></div></div></div></html>

`

Reply:
@jsnellbaker @robertrmartinez this feature needs to go last week but we missed the deadline pls let me know if u have any other questions and pls review as priority

Reply:
Ok, I still am not too sure about the use case, seems pretty unique to want banner in instream ad units.

But It looks fine code wise, Will let @jsnellbaker give it one last review and we can merge it before the release later today.

Reply:
Thanks, @robertrmartinez for the approval the use case is the custom integration between video ssp and one mobile. @jsnellbaker review pls